sdk/communication/Azure.Communication.MediaComposition/src/Models/ParticipantInput.cs

```diff
@@ -9,5 +10,33 @@ namespace Azure.Communication.MediaComposition.Models
     [CodeGenModel("Participant")]
     public partial class ParticipantInput
     {
+        [CodeGenMember("Id")]
+        internal CommunicationIdentifierModel ServiceId { get; set; }
+
+        /// <summary> The CommunicationIdentifier that identifies the participant.</summary>
+        public CommunicationIdentifier Id => CommunicationIdentifierSerializer.Deserialize(ServiceId);
+
+        internal ParticipantInput(CommunicationIdentifierModel serviceId, string call)
```

Comment:
This is copying over the generated constructor, which isn't ideal if the swagger changes significantly (like adding a new property). This was the most straightforward I could come up with to make the constructor internal, looking at instructions: https://github.com/Azure/autorest.csharp#replace-any-generated-member

Maybe there is a better way.